.rubocop.yml

```diff
@@ -102,9 +102,6 @@ Layout/DotPosition:
 Style/SingleLineBlockParams:
   Enabled: false
 
-Style/TrailingCommaInLiteral:
```

Comment:
Rubocop complained about this, but removing it through no issues. Trying to get back to as vanilla Rubocop config as possible.